src/components/20-molecules/file-upload/demo.js

```diff
@@ -37,7 +37,9 @@ storiesOf('Examples/File Upload/Pure HTML', module)
 
     const template = html`
       <div style="width:455px;">
-        <axa-file-upload class="js-file-upload__file-upload"
+        <axa-file-upload
+          maxSizeOfSingleFileKB="500"
```

Comment:
This is just that not all files are displayed with an error message (default is 100kb). The number is indeed random.

Reply:
Then why not 99999999?

Reply:
Because then no file would be invalid.

Reply:
But does the file need to be invalid at 500? I mean, if this is just a random number, then no file should ever become invalid right?

Reply:
Why did I changed it to 500kb? Because most of my files are bigger than 100kb and when I want to test I also want to test with valid files.
Why haven't I set it to 99999999kb? Because then all files would have a valid size, but I want to test both valid and ivalid ones...

And also it makes no sense to set it to 99999999kb because i can't imagine a use case to allow a 100gb file.

Reply:
Oh god... I totally missed that this is a demo and not the index.js file, sorry.